### DISCUSSION TITLE
How to set max/current players/motd?

### DISCUSSION THREAD

**Discussion by TheSilentPro** [Minestom/Minestom#1657] [Help] [answered]
how do i set the max/current players of the server?
how do i set the motd of the server?
i can't seem to find anything related to this in the docs

**Comment by mworzala** [comment, marked as answer]
You can set the ping response in the [ServerListPingEvent](https://javadoc.minestom.net/net/minestom/server/event/server/ServerListPingEvent.html). Though the max players set there is not enforced, that would have to be done by you.

**Comment by emortaldev** [comment]
https://github.com/Minestom/Minestom/discussions/822 motd part was answered here
